src/_posts/compliance/security-overview/2000-01-01-compliance.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Compliance
+nav: Compliance
+modified_at: 2024-06-20 00:00:00
+tags: compliance security measures
+index: 15
+---
+
+### GDPR
+
+Scalingo is compliant with the General Data Protection Regulation (GDPR) and we take all necessary measures to protect
+the privacy of our users' data. We do not share our users' data with third parties without their consent and we take all
+necessary measures to protect the data from unauthorized access. See our Data Processing Agreement
+[here](https://scalingo.com/data-processing-agreement).
+
+### Health Data Hosting (HDS)
+
+Scalingo is certified to host health data (HDS) in France. We have implemented the necessary security measures to
+protect the health data hosted on the platform and we are committed to protecting the privacy of the health data. See
+our HDS certification [here](/compliance/hds). You can see our Health Data Hosting Appendix
+[here](https://scalingo.com/gtc-appendix-health-data-hosting).
+
+### ISO 27001
+
+Scalingo is certified ISO 27001 and we have implemented a high level of security to protect the data hosted on the
+platform. The ISO 27001 certification ensures that we have implemented a comprehensive information security management
+system to protect the data hosted on the platform. See our ISO 27001 certification [here](/compliance/iso-27001).
+
+### SecNumCloud
+
+Scalingo is hosting its infrastructure in the SecNumCloud certified datacenters (uniquely for the region
+`osc-secnum-fr1`). This certification ensures that the datacenters have implemented a high level of security to protect
+the data hosted on them. We are working on a SecNumCloud certification for a new range of products.
```

Comment:
Should we mentioned someone the "Sovereignty" criteria ? Like "[...] _high level of security_ and guaranteeing a high level of technical, operational and legal standards. On the one hand, we offer a cloud computing service with a high level of IT hygiene, and on the other, data is protected in compliance with European law."

Reply:
Not at the moment